tests/test_Plant.py

```diff
@@ -0,0 +1,49 @@
+""" 
+    Tests the file Plant.py
+    Focuses on
+        -objects having the right type
+        -functions returning the correct value
+        -functions setting the correct value
+"""
+import src.Plant as p
```

Comment:
`import src.Plant.Plant` should be what you need to be able to just instantiate it as `item = Plant()` instead of the `p.Plant()` structure.

Reply:
For whatever reason, changing `import src.Plant` to `import src.Plant.Plant` causes an import error. I instead tried `from src.Plant import Plant` and this fixes the issue as well as keeping the tests passing. Honestly, I don't really understand the discrepancy between these two.

Reply:
Python imports are kind of hard to wrap your head around, to be honest. I actually sparked a day-long discussion on the ins and outs of the inner workings of Python's import system with my professional group, simply by asking how it works. In short, "it's complicated." 😆 

The basic rule of thumb (well, one of them) is that `import x` stops at the file level. If you want to import an entity inside a file, use the `from x import y` format.